src/main/java/trackitnus/model/ModelManager.java

```diff
@@ -156,6 +157,12 @@ public boolean hasModule(Module task) {
         return trackIter.hasModule(task);
     }
 
+    @Override
+    public boolean hasModule(Code code) {
+        requireNonNull(code);
+        return trackIter.hasModule(new Module(code, new Name("dummy")));
+    }
+
```

Comment:
This works? `hasModule` only checks based on code and not name?

Reply:
Actually the current hasModule is doing exactly this 😆 two modules are consider "same" if they have the same code